server/models/data/availabilitydetails.js

```diff
@@ -0,0 +1,19 @@
+const moment = require('moment');
```

Comment:
Hi Nancy, 
It does not appear that moment is being used anywhere in this file. Thus, I would recommend removing this library import to reduce unneeded code.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've removed `const moment = require('moment')` in server/models/data/availabilitydetails.js.

Please confirm if this takes care of your suggestion!